cli/utils.py

```diff
@@ -51,7 +51,8 @@ def parse_url(url: Optional[str]) -> Optional[URL]:
     if url is None:
         return None
 
-    burl = URL(url)
+    # We remove the path since we rely on it being empty when we append other paths in the program
```

Comment:
I think we should write:
We remove *the last slash* in the path since we rely on it being empty when we append other paths in the program